src/plots/cartesian/constants.js

```diff
@@ -0,0 +1,41 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+module.exports = {
+    // ms between first mousedown and 2nd mouseup to constitute dblclick...
+    // we don't seem to have access to the system setting
+    DBLCLICKDELAY: 600,
+
+    // pixels to move mouse before you stop clamping to starting point
+    MINDRAG: 8,
```

Comment:
OK, I've made a MINSELECT that's a little bigger than MINDRAG (12 instead of 8). https://github.com/plotly/plotly.js/commit/b5c090f2146a90f764cea1b607993b04c6148992

I don't want to make it as big as MINZOOM (or make it an increasing function of distance) because this limits how thin a rectangle you can select. Which is not a problem with MINZOOM because you can always zoom a second time.